javascript-create-module/pom.xml

```diff
@@ -74,6 +73,36 @@
                             <arguments>test</arguments>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>set-alpha-version</id>
+                        <phase>deploy</phase>
+                        <goals>
+                            <goal>yarn</goal>
+                        </goals>
+                        <configuration>
+                            <arguments>node ../.m2/sync-version.js alpha-${parsedVersion.majorVersion}.${parsedVersion.minorVersion}.${parsedVersion.incrementalVersion}-${timestamp}</arguments>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>publish-alpha-tag</id>
+                        <phase>deploy</phase>
+                        <goals>
+                            <goal>yarn</goal>
+                        </goals>
+                        <configuration>
+                            <arguments>npm publish --tag alpha --access public</arguments>
```

Comment:
Wondering if want more granularity with an alpha tag specific to the version (example: `0.7.0-alpha`) in addition to the generic `alpha` one.

Reply:
Discussed with @jkevan , we're OK with using this `"alpha"` tag for now